docs/logic-apps/logicapp-authentication.md

```diff
@@ -0,0 +1,25 @@
+[home](../README.md)
```

Comment:
Nit - Let's rename this file to `docs/logic-apps/authentication.md`

Reply:
Yes, of course.